UndertaleModTool/MainWindow.xaml.cs

```diff
@@ -715,10 +890,10 @@ private async Task LoadFile(string filename, bool preventClose = false)
                         #pragma warning disable CA1416
                         UndertaleCodeEditor.gettext = null;
                         UndertaleCodeEditor.gettextJSON = null;
+                        OpenInNewTab(new DescriptionView("Welcome to UndertaleModTool!", "Double click on the items on the left to view them!"), "Welcome!");
+                        ((Tab)CurrentTab).AutoClose = true;
+                        ((Tab)CurrentTab).SelectionHistory.Clear();
                         #pragma warning restore CA1416
```

Comment:
It should be moved to line 893 (right after `UndertaleCodeEditor.gettextJSON = null;`).